### DISCUSSION TITLE
How to run the program on kobo e-book reader?

### DISCUSSION THREAD

**Discussion by asuplive** [filebrowser/filebrowser#1937] [Q&A]
https://bookfere.com/post/823.html
Please see the link above, filebrowser works fine on kindle, now I've switched to kobo ebook reader but don't know how to run this program

Currently I have only two software installed on my kobo e-book reader, they are NickelMenu and koreader
https://github.com/pgaskin/NickelMenu
https://github.com/koreader/koreader/wiki

**Comment by shuvashish76** [comment]
See this plugin for KOReader: https://github.com/patelneeraj/filebrowserplus.koplugin/
